backend/src/main/java/gov/cdc/usds/simplereport/service/DeviceTypeService.java

```diff
@@ -183,6 +204,9 @@ public DeviceType createDeviceType(CreateDeviceType createDevice) {
         .map(specimenType -> new DeviceSpecimenType(dt, specimenType))
         .forEach(_deviceSpecimenRepo::save);
 
+    dt.setSupportedDiseases(supportedDiseases);
+    _repo.save(dt);
+
```

Comment:
another non-blocking question:
is there a reason we implement setting supported diseases differently than swab types? what affects the decision to create a repo like DeviceSpecimenTypeRepository instead of just using setSwabTypes?

Reply:
That is a great question! Zedd would probably have more insight since he wrote the code around swab types - there may have been some reason we needed a repository for the join table in that case that I'm not aware of. But since we don't really need to directly access the join table in this case, it's easier/simpler to just let Hibernate handle creating all the needed records for use.